service/plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/command/model/CommandResult.java

```diff
@@ -16,5 +16,4 @@ public class CommandResult {
     private JobStatus status;
     private Map<String, Object> result;
```

Comment:
Do we need this `result` attribute? If yes, where is it being set? I noticed it was always null in examples you provided in this thread https://solacedotcom.slack.com/archives/C060GEFNDM0/p1701207773149949

Reply:
I added the `result` for a potential future discovery usecase. Terraform would give us a really easy way to get all configurable resources on the broker. I wanted to add it now so we wouldn't have to iterate on the API in the future, but I realize that is not an agile way of operating. I'm okay with removing this until needed.

Reply:
Let's keep it if you say terraform can return results as response of certain TF commands